api/src/main/java/org/pmiops/workbench/api/UserController.java

```diff
@@ -86,11 +89,10 @@ public UserController(
    * @return
    */
   @Override
+  @Deprecated
   public ResponseEntity<UserResponse> user(
       String term, String pageToken, Integer pageSize, String sortOrder) {
-    UserResponse response = new UserResponse();
-    response.setUsers(Collections.emptyList());
-    response.setNextPageToken("");
+    UserResponse response = initializeUserResponse();
 
     if (null == term || term.isEmpty()) {
```

Comment:
Found this wired line while reviewing PR:
`null == term || term.isEmpty()`
Can you help change here to Guava's `Strings.isNullOrEmpty`
Thanks